contrib/automation_tests/core/common_controls.py

```diff
@@ -96,3 +101,23 @@ def get_item_at(self, row: int, col: int) -> BaseWrapper:
 
     def find_first_item_row(self, text: str, column: int, partial_match=False) -> int:
         return self._table_obj.find_first_item_row(text, column, partial_match)
+
+
+def get_tooltip(app: Application, control: BaseWrapper) -> str:
+    old_window_handle = app.top_window().handle
+
+    rectangle = control.rectangle()
+    position = (rectangle.left + floor((rectangle.right - rectangle.left) / 2),
+                rectangle.top + floor((rectangle.bottom - rectangle.top) / 2))
+
+    control.set_focus()
+    mouse.move(position)
+    mouse.move((position[0] + 1, position[1]))
+    time.sleep(2)
```

Comment:
Any chance to make this a "wait"?

Reply:
Hm, I could turn this into a wait. But I found the sleep() appropriate in this case since it's expected behavior of a tooltip that you need to hover for a short amount of time until it appears - the sleep() here isn't to hide timing issues, but "intended behavior". WDYT?

Reply:
Yeah, right, agree, makes sense here.